files/usr/share/cinnamon/applets/menu@cinnamon.org/applet.js

```diff
@@ -1150,7 +1150,7 @@ MyApplet.prototype = {
         this.settings.bind("activate-on-hover", "activateOnHover", this._updateActivateOnHover);
         this._updateActivateOnHover();
 
-        this.menu.actor.add_style_class_name('menu-background');
+        this.menu.setCustomStyleClass('menu-background');
```

Comment:
@collinss It'd be nice if all applets added a predictable custom style class to their menus. It won't happen overnight but I think this PR is a good place to start. What do you think?
```javascript
  this.menu.setCustomStyleClass('menu-background menu-for-applet--menu');
```

Then eventually...
```javascript
 /* Window List Applet */
  this.rightClickMenu.setCustomStyleClass('menu-for-applet--windowlist');
```
```javascript
 /* Sound Applet */
  this.menu.setCustomStyleClass('menu-for-applet--sound');
```